boxsdk/object/file.py

```diff
@@ -1,10 +1,9 @@
 # coding: utf-8
 
 from __future__ import unicode_literals
-
 from boxsdk.config import API
-
-from .item import Item
+from boxsdk.object.item import Item
```

Comment:
I think we should at least be consistent about if we are going to import with full path or relative path.